doc/content/guides/connecting-gateways/nasysoutdoorgateway/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "NASys LoRaWAN Outdoor Gateway"
+description: ""
+weight: 1
+---
+
+NASys LoRaWAN Outdoor Gateway is an 8 Channel LoRaWAN gateway, whose technical specifications can be found in [the official product page](https://www.nasys.no/product/lorawan-gateway/). This page guides you to connect it to {{% tts %}}.
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways.
+2. NASys LoRaWAN Outdoor Gateway connected to the internet (or your local network) via ethernet.
+
+## Registration
+
+Create a gateway by following the instructions for the [Console]({{< ref "/guides/getting-started/console#create-gateway" >}}) or the [CLI]({{< ref "/guides/getting-started/cli#create-gateway" >}}). Typically, the **EUI** field for your gateway should exist on the sticker at the bottom.
+
+## Configuration using a Terminal
+
+Find the IP address the gateway. This can be done in various ways. You can connect your machine to the same local network as that of the gateway Ethernet connection and scan for open SSH ports or assign a static IP to the gateway and use that. Once the gateway IP address is found, ssh into it.
+
+```bash
+$ ssh root@<GatewayIP>
+```
+
+The default username is **root**, and the default password can be also found in the sticker.
+
+Your gateway should come with a slightly modified version of the [Lora-net UDP packet forwarder](https://github.com/Lora-net/packet_forwarder) pre-installed at `/opt/nas-lgw`. There are two configuration files `global_conf.json` and `local_conf.json`, both located in `/opt/nas_lgw`.
+
+Typically, you should not need to touch the `global_conf.json` file. Instead, edit `local_conf.json`
```

Comment:
So where does the frequency plan come from?

Reply:
Will update the text to mention that the frequency plan configuration can be retrieved from GCS